docs/src/numerical_implementation/poisson_solvers.md

```diff
@@ -171,3 +175,24 @@ permutation of \eqref{eq:permutation} must be applied.
 Due to the extra steps involved in calculating the cosine transform in 2D, running with two wall-bounded dimensions
 typically slows the model down by a factor of 2. Switching to the FACR algorithm may help here as a 2D cosine transform
 won't be necessary anymore.
+
+## Iterative Solvers
```

Comment:
💯

When PRs also update the docs

Reply:
@ali-ramadhan - quick question. I can't find the updated docs! I was going to do a bit more and fixed so they build, but I can't figure out where to click to see the PR version!

Reply:
@christophernhill Ah I haven't figured out a way of putting the doc preview link in the PR so we just use the direct link generated.

For this PR it should be https://clima.github.io/OceananigansDocumentation/previews/PR1418/